src/databaseTest/java/net/sf/jabref/shared/DBMSConnectorTest.java

```diff
@@ -26,12 +27,9 @@
     public void testGetNewConnection() throws ClassNotFoundException, SQLException {
         DBMSConnectionProperties properties = TestConnector.getConnectionProperties(dbmsType);
 
-        try {
+        AssertUtil.assertDoesNotThrow(() -> {
```

Comment:
ItelliJ shows orange icon when unexpected Exception is thrown, but red icon when `assertDoesNotThrow` is used. I really like this triple state: failed, not failed, something unexpected happened (unexpected in the sense that the test does not forsee the exception). 

The old code did this with try/catch and Assert.fail(). I don't like that, because I don't see the exception clearly. With the code above, the Assertion fails and the exception is shown clearly.